syntax = "proto3";

package hellowrld;


service RaftServer {
  rpc RequestVote (VoteRequest) returns (VoteReply) {}
}

// The request message containing the user's name.
message VoteRequest {
  int32 id = 1;
  int32 term = 2;
}

// The response message containing the greetings
message VoteReply {
  optional int32 return_term = 1;
  bool voteGranted = 2;
}
